handlers/GameHandler.java

```diff
@@ -0,0 +1,68 @@
+package com.kbh.snl.handlers;
+
+import com.kbh.snl.models.Game;
+import com.kbh.snl.models.IntermediateObject;
+import com.kbh.snl.models.Ladder;
+import com.kbh.snl.models.Player;
+import com.kbh.snl.models.Snake;
+import java.util.Iterator;
+import java.util.Random;
+
+public class GameHandler {
+
+  private final Game game;
+  private final Random randomNumberGenerator;
+
+  public GameHandler(final Game game) {
+    this.game = game;
+    this.randomNumberGenerator = new Random();
+  }
+
+  public void rungame() {
+    while (game.getPlayersLeft().size() > 1) {
+      final Iterator<Player> playerIterator = game.getPlayersLeft().iterator();
+      executeNextStep(playerIterator);
+
+    }
+    System.out.println(String.format("%s wins the game", game.getSortedWinners()));
+  }
+
+  private void executeNextStep(Iterator<Player> playerIterator) {
+    while (playerIterator.hasNext()) {
+      final Player currentPlayer = playerIterator.next();
+      //throw dice
```

Comment:
Avoid comments. Make sure that your code is enough to understand what each line is doing.

Reply:
noted.